avaframe/runScripts/runRangeTimeDiagram.py

```diff
@@ -78,6 +80,15 @@
     # fetch dem data
     demInputs = gI.readDEM(avalancheDir)
 
+    # fetch all flow parameter result fields
+    configDir = pathlib.Path(avalancheDir, 'Outputs', 'com1DFA', 'configurationFiles')
+    if (configDir.is_dir() is False):
+        fU.fileNotFoundMessage(('No configuration files found in %s - consider first running avalanche simulations' %
+            configDir))
+    elif len(list(configDir.glob('*.ini'))) == 0:
+        fU.fileNotFoundMessage(('No configuration files found in %s - consider first running avalanche simulations' %
+            configDir))
```

Comment:
Twice exactly the same lines, shouldn't the checking be as simple or?

Reply:
true